pype_schema/utils.py

```diff
@@ -491,6 +492,8 @@ class PumpType(Enum):
 
     Constant = auto()
     VFD = auto()
+    ERI = auto()
```

Comment:
I am changing ERI to ERD (energy recovery device) throughout as I believe this is a typo. Let me know if that's incorrect in some way